Build/GetWix.ps1

```diff
@@ -0,0 +1,21 @@
+function GetWix {

```

Comment:
We don't have this code anywhere else, this is new. Also no reason why it couldn't be building without .NET 4.5.